CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Release notes for Miranj's Craft Boilerplate starter project.
 - Added [Alpine.js](https://alpinejs.dev/), alongwith the [Collapse plugin](https://alpinejs.dev/plugins/collapse).
 - Added icons for some common podcast hosts.
 - Fixed a bug where arbitrary value Tailwind classes with a `,` would always be purged.
+- Fixed an issue where images with missing dimensions were causing a page loading failure.
```

Comment:
This should go under _Craft_, not _Front-end_ because it is a server-side bug.

Reply:
Okay.